composites/Plugin/DashboardWidget/components/Divider.js

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+import colors from "../../../../style-guide/colors";
+
+/**
+ * Simple divider
+ */
+const DashboardWidgetDivider = styled.div`
+	box-sizing: border-box;
+	height: 1px;
+	border-top: 1px solid ${ colors.$palette_grey };
+`;
```

Comment:
Note: Using markup to generate a horizontal border it's a bit against separation of content and presentation but I won't bother too much about this. 😬